src/reader/widgets/ReaderWidget.vue

```diff
@@ -24,6 +24,52 @@
             :textWidth="textWidth"
           />
         </template>
+        <Alignments
+          v-else-if="alignmentMode"
+          :alignments="alignments"
+          :textSize="textSize"
+          :textWidth="textWidth"
+        />
+        <div
+          v-else-if="namedEntitiesMode"
+          class="entity-mode"
+          :class="`map-direction-${showMap ? showMap : 'none'}`"
+        >
+          <div class="toolbar">
```

Comment:
There can be some overflow issues with this toolbar when scrolling:

https://deploy-preview-42--explorehomer-dev.netlify.app/reader?urn=urn%3Acts%3AgreekLit%3Atlg0012.tlg001.perseus-eng4%3A2.480-2.520

![image](https://user-images.githubusercontent.com/629062/86180403-aec12680-baf1-11ea-8c99-061aac32b101.png)

Reply:
It also feels like it needs to be its own component to avoid cluttering `ReaderWidget` (e.g. MapToolbar).

Reply:
Done